packages/workspace/src/utilities/typescript.ts

```diff
@@ -36,7 +36,7 @@ function readTsConfigOptions(tsConfigPath: string) {
     tsModule.sys.readFile
   );
   // we don't need to scan the files, we only care about options
-  const host = {
+  const host: any = {
```

Comment:
Could we use `Partial<ts.ParseConfigHost>` rather than any here?

Reply:
Yeah, we can do that, however the call to `parseJsonConfigFileContent` errors due to it not being the same type.

We can go half-way and improve the type safety a bit doing so:


```ts
const host: Partial<ts.ParseConfigHost> = {
    readDirectory: () => [],
    fileExists: tsModule.sys.fileExists,
  };
  return tsModule.parseJsonConfigFileContent(
    readResult.config,
    host as any,
    dirname(tsConfigPath)
  ).options;
```

Reply:
A bit nitpicky, but I think in your halfway solution it would let you case to ts.ParseConfigHost instead of any during the call

Reply:
A bit nitpicky, but I think in your halfway solution it would let you case to ts.ParseConfigHost instead of any during the call

Reply:
As in

```ts
  const host: Partial<ts.ParseConfigHost> = {
    readDirectory: () => [],
    fileExists: tsModule.sys.fileExists,
  };
  return tsModule.parseJsonConfigFileContent(
    readResult.config,
    host as ts.ParseConfigHost,
    dirname(tsConfigPath)
  ).options;
```

Reply:
Yeah